### PR TITLE
Dev bedclassifier script

### PR DESCRIPTION
**Pull request by donaldcampbelljr**
- Add bed classifier script which fetches bed files, classifies them and then reports the types to pephub. 
- Use this script to aid in tuning the bed classifier system.

### CODE REVIEW

**Comment by donaldcampbelljr** [comment]
More work towards #60 

The major modification to the pipeline:
- add exception handling around utf-16 files.

The script I added for testing runs separately from the main pipeline and should not impact performance.

Most recent testing added to PEP: https://pephub.databio.org/donaldcampbelljr/bedclassifier_tuning_geo?tag=default
 ~95% accuracy on classification for bed files for n=1567 bedfiles

File: scripts/bedclassifier_tuning/bedclassify.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+        all_values.update({"types_match": do_types_match})
+
+        try:
+            psm.report(record_identifier=bed_digest, values=all_values)
+        except Exception as e:
+            _LOGGER.warning(msg=f"FAILED {bed_digest}  Exception {e}")
+
+        if self.pm:
+            self.pm.stop_pipeline()
```
Comment (khoroshevskyi) [inline scripts/bedclassifier_tuning/bedclassify.py:114]: You are duplicating a lot of code here, just use main function from bedboss

Comment (khoroshevskyi) [reply scripts/bedclassifier_tuning/bedclassify.py:114]: I think this code should be deleted, because it makes mess for future work

**Review by khoroshevskyi — CHANGES_REQUESTED** [review]
Could you please clean just one script? 
Changes approved

**Comment by donaldcampbelljr** [comment]
We chatted about the above and agreed to merge the changes since only exceptions were added to main pipeline and everything else (such as the BedClassifier Class) are self contained

**Pull request merged**